deployments/rainbow-brass/src/main/java/org/sa/rainbow/brass/model/map/EnvMap.java

```diff
@@ -247,7 +249,10 @@ public synchronized void loadNodesFromFile(String mapFile){
         for (Object node : nodes) {
             JSONObject jsonNode = (JSONObject) node;
             String id = (String) jsonNode.get("node-id");
-            JSONObject src_coords = (JSONObject) jsonNode.get("coord");
+            JSONObject src_coords = (JSONObject) jsonNode.get("coords");
+            if (src_coords == null) { // Try backward compatible "coord"
```

Comment:
Is the data structure the same for "coord" and "coords"?

Reply:
This was for backwards compatibility. In Phase 1, we used "coord" in phase 2 somehow we shifted to "coords". I thought I would maintain both in case we have anything old hanging around (e.g., maybe in Pooyan's CP)

Reply:
I'm fine with backwards-compatible code, but it would be nice to minimize the incompatibility in map format between CPs 1 & 3.

Reply:
Oh yes. Don't get me wrong -- I think that he is using the mapeditor now, which generates the new one, so we should be ok.